front/src/applications/stdcm/components/StdcmForm/StdcmOperationalPoint.tsx

```diff
@@ -28,40 +26,68 @@ const StdcmOperationalPoint = ({ location, pathStepId, disabled }: StdcmOperatio
   const { t } = useTranslation('stdcm');
   const dispatch = useAppDispatch();
 
-  const { searchTerm, sortedSearchResults, setSearchTerm, setChCodeFilter } =
-    useSearchOperationalPoint({
-      initialSearchTerm: location?.name,
-      initialChCodeFilter: location?.secondary_code,
-    });
+  const {
+    searchTerm,
+    setSearchTerm,
+    sortedSearchResults: searchResults,
+    setSearchResults,
+  } = useSearchOperationalPoint({
+    initialSearchTerm: location?.name,
+    initialChCodeFilter: location?.secondary_code,
+  });
+
+  const [chSuggestions, setChSuggestions] = useState<{ label: string; id: string }[]>([]);
 
   const { updateStdcmPathStep } = useOsrdConfActions() as StdcmConfSliceActions;
 
-  const operationalPointsSuggestions = useMemo(
+  const selectedCi = useMemo(
+    () =>
+      location
+        ? {
+            ...location,
+            label: [location.trigram, location.name].join(' '),
+          }
+        : undefined,
+    [location]
+  );
+
+  const selectedCh = useMemo(
+    () =>
+      location
+        ? { label: formatChCode(location.secondary_code), id: location.secondary_code }
```

Comment:
I know it can't happen but since we do the test for label, we should also do it for the id, otherwise sometimes it will be an empty string
```suggestion
        ? { label: formatChCode(location.secondary_code), id:  formatChCode(location.secondary_code) }
```